ouroboros-network-framework/test/Test/Simulation/Network/Snocket.hs

```diff
@@ -338,30 +354,299 @@ toBearerInfo abi =
 -- Properties
 --
 
+data TestError = UnexpectedOutcome
```

Comment:
I think it's worth to define more constructors which are used depending on the situation.

Reply:
fixed!